api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -1020,6 +1020,16 @@ public class ApiConstants {
     public static final String PRIVATE_MTU = "privatemtu";
     public static final String MTU = "mtu";
     public static final String LIST_APIS = "listApis";
+    public static final String VM_SCHEDULE_DESCRIPTION = "description";
```

Comment:
VM_ prefix isn't necessary unless the parameter is specifically for VM Schedule feature, this for example can simply be description (it's possible the DESCRIPTION param already exists that you can simply reuse).

Reply:
Fixed

Reply:
Fixed

Reply:
@rahulbcn27 as others mentioned you can use existing constants instead of defining new ones